src/jit/flowgraph.cpp

```diff
@@ -9021,6 +9021,49 @@ void Compiler::fgSimpleLowering()
                     // Add in a call to an error routine.
                     fgSetRngChkTarget(tree, false);
                 }
+
+#if FEATURE_FIXED_OUT_ARGS
+                if (tree->gtOper == GT_CALL)
```

Comment:
Can you please (1) add a standard function header to this function that describes its purpose and components, and (2) write an overall comment for this new code that mentions *why* it is located here (e.g., "this happens to be a convenient place that follows all optimization that does a full IR pass")?